documentation/modules/appendix_crds.adoc

```diff
@@ -2792,42 +2792,46 @@ include::../api/io.strimzi.api.kafka.model.KafkaBridgeSpec.adoc[leveloffset=+1]
 
 [options="header"]
 |====
-|Property                 |Description
-|replicas          1.2+<.<a|The number of pods in the `Deployment`.
```

Comment:
This generated file ended up indented differently, not sure what I've done wrong

Reply:
It indents according to the longest field. So this is OK.